periodic-extent-spec.md

```diff
@@ -51,26 +51,16 @@ The object describes the temporal extents of the Collection.
 
 | Element  | Type               | Description                                                           |
 | -------- | ------------------ | --------------------------------------------------------------------- |
-| interval | \[\[string\|null] \| \[[Periodic Interval](#periodic-interval)]] | **REQUIRED.** Potential *temporal extents* covered by the Collection. |
+| interval | \[\[string\|null]] | **REQUIRED.** Potential *temporal extents* covered by the Collection. |
+| period   | string             | **OPTIONAL.** ISO 8601:1988(E) compilant period: PnYnMnD |
 
 **interval**: Each outer array element can be a separate temporal extent, but it is recommended to only use multiple temporal extents if a union of them would then include a large uncovered time span (e.g. only having data for the years 2000, 2010 and 2020).
+**period**: Is an optional field. Period is represented by ISO 8601:1988(E) compilant string.
```

Comment:
Is `period` optional? For compliance with this extension, it seems like `period` is required. I don't think the extension needs to model the union of the two possibilities, only to describe what's required for it to be valid. Or is it optional because of possibly plural intervals?

Reply:
@jisantuc I think it is optional because I modeled the union, I'll fix it, thanks!

Reply:
😎